### PR TITLE
Feature/booking

### PR DESCRIPTION
**Pull request by RafSobolQAC**

### CODE REVIEW

File: app/controllers/Controller.scala [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package controllers
+
+class Controller {
```
Comment (RafSobolQAC) [inline app/controllers/Controller.scala]: Codacy Issue found: [Redundant braces after class definition](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891&bid=17187718)

File: app/models/Booking.scala [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package models
+
+
+import java.time.LocalDateTime
+
+import akka.http.scaladsl.model.DateTime
+import play.api.libs.json.{Format, JsObject, JsResult, JsValue, Json, OFormat, Reads}
```
Comment (RafSobolQAC) [inline app/models/Booking.scala]: Codacy Issue found: [Use wildcards when selecting more than 6 elements](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 2
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891&bid=17187718)

File: app/controllers/Controller.scala (hunk 1 above)
Comment (RafSobolQAC) [inline app/controllers/Controller.scala]: Codacy Issue found: [Redundant braces after class definition](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

File: app/models/Booking.scala (hunk 2 above)
Comment (RafSobolQAC) [inline app/models/Booking.scala]: Codacy Issue found: [Use wildcards when selecting more than 6 elements](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891&bid=17187718)

File: app/controllers/Controller.scala [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
```
Comment (RafSobolQAC) [inline app/controllers/Controller.scala]: Codacy Issue found: [Redundant braces after class definition](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891&bid=17187718)

File: app/controllers/Controller.scala (hunk 3 above)
Comment (RafSobolQAC) [inline app/controllers/Controller.scala]: Codacy Issue found: [Redundant braces after class definition](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

**Review by dannywaughman — APPROVED** [review]
Looks very nice so far

File: app/models/JsonFormats.scala [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, LocalDateTime, LocalTime, ZoneOffset}
+import java.util.Date
+
+
+object JsonFormats {
+
+  import reactivemongo.bson.{BSONObjectID, BSONDateTime}
+  import play.api.libs.json.{Json, _}
```
Comment (RafSobolQAC) [inline app/models/JsonFormats.scala]: Codacy Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

**Comment by RafSobolQAC** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 4
           
```


See the complete overview on [Codacy](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891&bid=17187718)

File: app/models/Movie.scala [hunk 5]
```diff
@@ -0,0 +1,22 @@
...
+
+case class Movie(image: String,
+                 title: String,
+                 director: String,
+                 description: String,
+                 actors: List[String],
+                 var screenings: List[DateTime])
+
+case class MovieWithID(
+                        _id: BSONObjectID,
```
Comment (RafSobolQAC) [inline app/models/Movie.scala]: Codacy Issue found: [Field name '_id' does not match the regular expression '^[a-z][A-Za-z0-9]*$'](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

File: app/models/JsonFormats.scala [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+  import reactivemongo.play.json.BSONFormats._
```
Comment (RafSobolQAC) [inline app/models/JsonFormats.scala]: Codacy Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

File: app/models/JsonFormats.scala [hunk 7]
```diff
@@ -0,0 +1,22 @@
+package models
+
+import java.time.format.DateTimeFormatter
+import java.time.{Instant, LocalDateTime, LocalTime, ZoneOffset}
+import java.util.Date
+
+
+object JsonFormats {
+
+  import reactivemongo.bson.{BSONObjectID, BSONDateTime}
```
Comment (RafSobolQAC) [inline app/models/JsonFormats.scala]: Codacy Issue found: [Put imports in the beginning of the package](https://app.codacy.com/manual/RafSobolQAC/cinema-team-one/pullRequest?prid=5243891)

**Review by dannywaughman — APPROVED** [review]
Yep

**Pull request merged**